clients/src/main/java/org/apache/kafka/common/protocol/ProtoUtils.java

```diff
@@ -40,6 +40,12 @@ public static short latestVersion(int apiKey) {
         return Protocol.CURR_VERSION[apiKey];
     }
 
+    public static short oldestVersion(int apiKey) {
```

Comment:
Nit: `oldest` and `latest` are not symmetric. Also, it's a bit odd that we use different names for the field names (basically `current` and `minimum`). Finally, why do we use `int` instead of `ApiKeys`?